tests/test_auto_testing.py

```diff
@@ -96,3 +97,25 @@ def _predict(self, item):
 def test_testing_model_library(testing_model_library):
     m = testing_model_library.get("some_model")
     assert m({"x": 1}).x == 1
+
+
+@pytest.mark.parametrize(
```

Comment:
Could you possibly add a test of `kwargs` as well?

Reply:
sure !